docs/pages/services/cassandra/limitations.md

```diff
@@ -0,0 +1,292 @@
+---
+post_title: Limitations
+menu_order: 80
+feature_maturity: preview
+enterprise: 'no'
+---
+
+* Seed Node Replace
+
+While the Cassandra service supports a node replace operation via the command
+`dcos cassandra pods replace ≤node_id≥`, the node replace operation does not
+currently inform the underlying Cassandra cluster of the change, so effectively
```

Comment:
This isn't strictly true, `replace_address` nominally accomplishes this with non-seeds.